crud/update.lua

```diff
@@ -62,6 +62,12 @@ local function call_update_on_router(space_name, key, user_operations, opts)
         return nil, UpdateError:new("Wrong operations are specified: %s", err), true
     end
 
+    if type(key) == 'table' or type(key) == 'number' then
```

Comment:
Why "number"? space:get(...) works fine even with strings and other types. Seems this check is redundant.